recipe/meta.yaml

```
@@ -1,20 +1,20 @@
 {% set name = "qpd" %}
-{% set version = "0.4.1" %}
+{% set version = "0.4.4" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qpd-{{ version }}.tar.gz
-  sha256: 2bf3fb0a4cf2307b7f3df06ac6d7a4396da7867350d17f3c37cb415fbb8089f7
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e0ed05b88e321ea9935874377bda11339c90f1469f34344e9b41d16b8088e136
 
 build:
-  # s390x is missing triad  
-  skip: true  # [py<37 or s390x]
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
+  # s390x is missing triad  
+  skip: true  # [py<38 or s390x]
```

```suggestion
  skip: true  # [py<37 or s390x]
```
https://inspector.pypi.io/project/qpd/0.4.4/packages/81/51/82f1751e3aea61edf0c36e1bb8284e57f0856b550f9bc44be42d61335bf2/qpd-0.4.4.tar.gz/qpd-0.4.4/setup.py#line.58